src/App.js

```diff
@@ -1,27 +1,34 @@
-import logo from './logo.svg';
 import './App.css';
+import { v4 as uuidv4 } from 'uuid';
+import Calculator from './components/Calculator';
+import Screen from './components/Screen';
+import ButtonBox from './components/ButtonBox';
+import Button from './components/Button';
+
+const btnValues = [
+  ['C', '+/-', '%', '/'],
+  [7, 8, 9, 'X'],
+  [4, 5, 6, '-'],
+  [1, 2, 3, '+'],
+  [0, '.', '='],
+];
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator>
+        <Screen />
+        <ButtonBox>
+          {btnValues.map((row) => (
+            row.map((btn) => (
+              <Button
+                value={btn}
+                key={uuidv4()}
+              />
+            ))
+          ))}
+        </ButtonBox>
+      </Calculator>
     </div>
   );
```

Comment:
- [ ] According to the project requirement you should create the design in your `Calculator.js` file. So Please remove the buttons value in from your App.js and use in the `Calculator.js`  component. You just need to need to import `Calculator.js` component into `App.js`.
- [ ] Also make sure that Calculator.js have at least 1 child component.
### Requirement
![Screenshot 2023-09-14 225909](https://github.com/MohammadYaser/math-magicians/assets/81727988/02861be8-d9b6-413b-85ee-387fb8ba83dd)